Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -27,7 +27,7 @@ final class AttributeTests: XCTestCase {
         DiagnosticSpec(message: "expected ')' to end attribute", fixIts: ["insert ')'"]),
       ],
       fixedSource: """
-        @_dynamicReplacement(for: <#identifier#>)
+        @_dynamicReplacement(for : <#identifier#>)
```

Comment:
What's the whitespace for?

Reply:
Oh, I wanted to fix that. It’s because `for` is a keyword now and keywords always have trailing space in BasicFormat at the moment. Improving that is a slightly larger change in BasicFormat that I would like to do in a follow-up PR.

Reply:
Here we go: https://github.com/apple/swift-syntax/pull/1298